blockies/src/main/java/pm/gnosis/blockies/Blockies.kt

```diff
@@ -0,0 +1,94 @@
+package pm.gnosis.blockies
+
+import android.graphics.Color
+
+internal class Blockies(val primaryColor: Int,
+                        val backgroundColor: Int,
+                        val spotColor: Int,
+                        val data: DoubleArray) {
+    companion object {
+        const val SIZE = 8
+
+        fun fromAddress(address: String): Blockies? {
```

Comment:
Maybe it would be better to use a BigInteger here (or some Address object in the future :P ), so that we always have the correct format of the string